packaging/windows/BUILD.txt

```diff
@@ -0,0 +1,120 @@
+How to make a darktable windows installer (64 bit only):
+
+1). Install MSYS2 (instructions and prerequisites can be found on official website: https://msys2.github.io/).
+    
+    Install x64 developer toolchain from the MSYS2 prompt:
+    $ pacman -S mingw-w64-x86_64-toolchain
+    
+    Install required dependencies:
+    $ pacman -S mingw-w64-x86_64-{exiv2,lcms2,lensfun,libglade,dbus-glib,libusb-compat,openexr,sqlite3,libxslt,libsoup,libwebp,libsecret,lua,graphicsmagick,openjpeg2,gtk-engines,gtk3,pugixml,sdl,libexif}
+    
+    For osm-gps-map:
+    $ pacman -S mingw-w64-x86_64-{python2-cairo,python2-pygtk,osm-gps-map}
+
+    For gphoto2:
+    $ pacman -S mingw-w64-x86_64-{libusb,popt,libgphoto2}
+    Check that CAMLIBS and IOLIBS for LIBGPHOTO are available
+
+    For Saxon:
+    $ pacman -S unzip perl-Locale-Gettext perl-TermReadKey perl-sgmls
+
+    For usermanual 
+    $ pacman -S mingw-w64-x86_64-imagemagick
+
+    For other gtk libraries:
+    $ pacman -S mingw-w64-x86_64-{atkmm, glibmm, cairomm, gtkmm3, gtksourceview3}
+
+
+    Install SAXON:
+    https://sourceforge.net/projects/saxon/files/Saxon-HE/
+    copy files into /usr/share/java/saxon/
```

Comment:
I think as long as the CLASSPATH points to the right directory it can be anywhere I guess.